### PR TITLE
11585 pylint

### PR DESCRIPTION
**Pull request by AndreiSavici**
Reduce pylint warnings. This is ticket [#11585](http://trac.mantidproject.org/mantid/ticket/11585). Unit tests and systemtest should pass

### CODE REVIEW

File: Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py [hunk 1]
```diff
@@ -27,23 +28,21 @@ def summary(self):
...
 
         self.declareProperty(WorkspaceGroupProperty('CorrectionsWorkspace', '',
-                             optional=PropertyMode.Optional, direction=Direction.Input),
+                                                    optional=PropertyMode.Optional, direction=Direction.Input),
                              doc='Name for the input Corrections workspace.')
 
         self.declareProperty(MatrixWorkspaceProperty('CanWorkspace', '',
-                             optional=PropertyMode.Optional, direction=Direction.Input),
+                                                     optional=PropertyMode.Optional, direction=Direction.Input),
                              doc='Name for the input Can workspace.')
```
Comment (eXeC64) [inline Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py]: Perhaps `doc` should be indented to the same level as `optional` here, for consistency.

File: Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CreateMD.py [hunk 2]
```diff
@@ -110,24 +109,31 @@ def PyInit(self):
...
+                                                direction=Direction.Input ), doc='Lattice angles' )
 
-        self.declareProperty(FloatArrayProperty('u', values=[], validator=FloatArrayMandatoryValidator(), direction=Direction.Input ), doc='Lattice vector parallel to neutron beam' )
+        self.declareProperty(FloatArrayProperty('u', values=[], validator=FloatArrayMandatoryValidator(),
+                                                direction=Direction.Input ), doc='Lattice vector parallel to neutron beam' )
 
-        self.declareProperty(FloatArrayProperty('v', values=[], validator=FloatArrayMandatoryValidator(), direction=Direction.Input ), doc='Lattice vector perpendicular to neutron beam in the horizontal plane' )
+        self.declareProperty(FloatArrayProperty('v', values=[], validator=FloatArrayMandatoryValidator(),
+                                                direction=Direction.Input ),
+                             doc='Lattice vector perpendicular to neutron beam in the horizontal plane' )
```
Comment (eXeC64) [inline Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CreateMD.py]: There are also multiple indentation levels here.

File: Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py [hunk 3]
```diff
@@ -157,8 +157,9 @@ def category(self):
         return 'Diffraction;PythonAlgorithms'
 
     def summary(self):
-        return "This Python algorithm performs the operations necessary for the reduction of diffraction data from the Osiris instrument at ISIS \
-              into dSpacing, by correcting for the monitor and linking the various d-ranges together."
+        return "This Python algorithm performs the operations necessary for the reduction of diffraction data \
+                from the Osiris instrument at ISIS \
+                into dSpacing, by correcting for the monitor and linking the various d-ranges together."
```
Comment (eXeC64) [inline Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py]: Won't the indentation here end up being included in the error string? Maybe having a separate string on each line, and joining them with a `+` would be better.

File: Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py [hunk 4]
```diff
@@ -166,9 +167,11 @@ def PyInit(self):
         self.declareProperty('Sample', '', doc=runs_desc)
         self.declareProperty('Vanadium', '', doc=runs_desc)
         self.declareProperty(FileProperty('CalFile', '', action=FileAction.Load),
-                             doc='Filename of the .cal file to use in the [[AlignDetectors]] and [[DiffractionFocussing]] child algorithms.')
+                             doc='Filename of the .cal file to use in the [[AlignDetectors]] and \
+                                  [[DiffractionFocussing]] child algorithms.')
         self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '', Direction.Output),
-                             doc="Name to give the output workspace. If no name is provided, one will be generated based on the run numbers.")
+                             doc="Name to give the output workspace. If no name is provided, \
+                                 one will be generated based on the run numbers.")
```
Comment (eXeC64) [inline Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py]: Same comment as above.

File: Code/Mantid/scripts/Inelastic/Direct/NonIDF_Properties.py [hunk 5]
```diff
@@ -96,7 +96,7 @@ def log(self, msg,level="notice"):
                          If not explicitly set, white beam for sample run is used.""")
     # TODO: do something about it.  Second white is explicitly used in
     # diagnostics but not accessed at all
-    second_white  = RunDescriptor("""Second white beam run resutlts currently unused in the workflow 
+    second_white  = RunDescriptor("""Second white beam run resutlts currently unused in the workflow
                     despite being referred to in Diagnostics.
                     In a future it should be enabled.""")
     #
```
Comment (eXeC64) [inline Code/Mantid/scripts/Inelastic/Direct/NonIDF_Properties.py]: Same thing as before, is the indentation wanted in the middle of this string?

File: Code/Mantid/scripts/PyChop.py [hunk 6]
```diff
@@ -1,4 +1,4 @@
-#pylint: disable=invalid-name
+#pylint: #pylint: skip-file
```
Comment (eXeC64) [inline Code/Mantid/scripts/PyChop.py]: I think this may be a typo.

**Comment by eXeC64** [comment]
Other than the comments I raised above (the indentation in string thing happens several more times) this looks good.

**Comment by AndreiSavici** [comment]
The indentation has to match the outer parenthesis. So if I have abc(def(alpha,beta,gamma),doc='test'), I need to indent

abc(defg(alpha,
              beta,
              gamma),
       doc='test')

**Comment by eXeC64** [comment]
Ah, that makes sense. I hadn't noticed the parenthesise.

**Pull request merged**